Modules/_uuidmodule.c

```diff
@@ -91,45 +111,1649 @@ py_windows_has_stable_node(void)
+    // UUID fields in little-endian order need to be byte-swapped:
+    // - time_low (4 bytes)
+    // - time_mid (2 bytes)
+    // - time_hi_version (2 bytes)
+    // - clock_seq_hi_variant (1 byte) - no swap needed
+    // - clock_seq_low (1 byte) - no swap needed
+    // - node (6 bytes) - no swap needed
+
+    unsigned char *src = (unsigned char *)bytes_le->buf;
+    unsigned char *dst = (unsigned char *)self->bytes;
+
+    // Swap time_low (bytes 0-3)
+    dst[0] = src[3];
+    dst[1] = src[2];
+    dst[2] = src[1];
+    dst[3] = src[0];
+
+    // Swap time_mid (bytes 4-5)
+    dst[4] = src[5];
+    dst[5] = src[4];
+
+    // Swap time_hi_version (bytes 6-7)
+    dst[6] = src[7];
+    dst[7] = src[6];
+
+    // Copy clock_seq and node as-is (bytes 8-15)
+    memcpy(dst + 8, src + 8, 8);
+
+    return 0;
+}
+
+static int
+validate_int(uuid_state *state, PyObject *int_value)
+{
+    if (!PyLong_Check(int_value)) {
+        PyErr_SetString(PyExc_TypeError, "value must be an integer");
+        return -1;
+    }
+
+    int cmp = PyLong_IsNegative(int_value);
+    if (cmp < 0) {
+        return -1;
+    }
+    if (cmp == 1) {
+        PyErr_SetString(PyExc_ValueError,
+            "int is out of range (need a 128-bit value)");
+        return -1;
+    }
+
+    // Check if it's greater than max (2^128 - 1)
+    cmp = PyObject_RichCompareBool(int_value, state->uint128_max, Py_GT);
+    if (cmp < 0) {
+        return -1;
+    }
+    if (cmp == 1) {
+        PyErr_SetString(PyExc_ValueError,
+            "int is out of range (need a 128-bit value)");
+        return -1;
+    }
+
+    return 0;
+}
+
+static int
+from_int(uuidobject *self, PyObject *int_value, int validate)
+{
+    // Convert a 128-bit integer to UUID bytes (big-endian)
+
+    uuid_state *state = get_uuid_state_by_cls(Py_TYPE(self));
+
+    if (validate && validate_int(state, int_value) < 0) {
+        return -1;
+    }
+
+    if (_PyLong_AsByteArray(
+            (PyLongObject *)int_value,
+            (unsigned char *)self->bytes,
+            16,
+            0,  // big-endian
+            0,  // unsigned
+            1   // with_exceptions
+        ) < 0)
+    {
+        return -1;
+    }
+
+    return 0;
+}
+
+static int
+extract_field(
+    PyObject *fields,
+    int field_num,
+    uint64_t max_value,
+    const char *error_msg,
+    uint64_t *result
+) {
+    PyObject *field = PySequence_GetItem(fields, field_num);
+    if (field == NULL) {
+        return -1;
+    }
+
+    if (!PyLong_Check(field)) {
+        PyErr_Format(PyExc_TypeError, "field %d must be an integer", field_num);
+        goto fail;
+    }
+
+    int overflow;
+    uint64_t value = PyLong_AsLongLongAndOverflow(field, &overflow);
+    if (overflow || (value == (uint64_t)-1 && PyErr_Occurred())) {
+        PyErr_Format(PyExc_ValueError, "%s", error_msg);
+        goto fail;
+    }
+
+    if (value > max_value) {
+        PyErr_Format(PyExc_ValueError, "%s", error_msg);
```

Comment:
Just use PyErr_SetString instead

Reply:
good catch; I used err_format for debug purposes and forgot to roll it back